easybuild/easyconfigs/p/pFUnit/pFUnit-4.2.0-iimpi-2021a.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'pFUnit'
+version = '4.2.0'
+local_fargparse_major_minor_version = '1.1'
+local_gftl_major_minor_version = '1.3'
+local_gftl_shared_major_minor_version = '1.2'
+
+homepage = 'https://github.com/Goddard-Fortran-Ecosystem/pFUnit'
+description = """pFUnit is a unit testing framework enabling JUnit-like testing of serial
+ and MPI-parallel software written in Fortran."""
+
+toolchain = {'name': 'iimpi', 'version': '2021a'}
+
+source_urls = ['https://github.com/Goddard-Fortran-Ecosystem/pFUnit/releases/download/v%(version)s']
+sources = [SOURCE_TAR]
+checksums = ['33df62f80cf03827455508b67d53f820ddffa2ec0f1ba999790ff1f87592ce16']
+
+builddependencies = [
+    ('CMake', '3.20.1', '', ('GCCcore', '10.3.0')),
+]
+
+dependencies = [
+    ('Python', '3.9.5', '', ('GCCcore', '10.3.0')),
+]
```

no need to make the toolchains explicit here, they'll get automatically mapped
```suggestion
builddependencies = [
    ('CMake', '3.20.1'),
]

dependencies = [
    ('Python', '3.9.5'),
]
```